lib/game.rb

```diff
@@ -0,0 +1,64 @@
+class Game
+    attr_reader :players, :board, :current_player, :other_player
+
+    def initialize(players, board = Board.new)
+      @players = players
+      @board = board
+      @current_player, @other_player = players.shuffle
+    end
+
+    def switch_players
+      @current_player, @other_player = @other_player, @current_player
+    end
+
+    def solicit_move
+      "#{current_player.name}: Enter a number between 1 and 9 to make your move"
+    end
+
+    def get_move(human_move = gets.chomp)
```

Comment:
Game logic should be kept in classes (no “puts” and “get.chomp” inside the `game.rb`)